src/apps/core/views/monster-key-value-editor-row.html

```diff
@@ -0,0 +1,9 @@
+<div class="monster-key-value-editor-row">
+	<div class="data-key">
+		<input required id="{{inputName}}.key" name="{{inputName}}.key" type="text" placeholder="{{coalesce keyPlaceholder i18n.keyValueEditor.keyPlaceholder}}" value="{{key}}"></input>
+	</div>
+	<div class="data-value">
+		<input required id="{{inputName}}.value" name="{{inputName}}.value" type="text" placeholder="{{coalesce valuePlaceholder i18n.keyValueEditor.valuePlaceholder}}" value="{{value}}"></input>
+	</div>
+	<i class="fa fa-trash-o fa-lg monster-red" data-action="remove"></i>
```

Comment:
Actionable icons should be wrapped into `a` tags for accessibility purposes.

Also, maybe use a `telicon@2` icon here.

The `data-action` seems overkill, you can use a class on the `a` tag once implemented.

Lastly, the repetition of those icons looks heavy so having them defined with `visibility: hidden` by default and only shown on row hover should be easier on the eye.